### ISSUE TITLE
Script Failure does not provide indication in script_success metric as to what script failed

### ISSUE THREAD

**Issue by chrisfw** [ricoberger/script_exporter#21]
Hi,
When a script fails, I won't be able to correlate back to the script that actually failed.  It would be great if it was included as a dimension in the script_success metric.  Alternatively or additionally, it would also be great if the output parameter was respected as in the case of my script I output an up/down metric that I can build my alerting off of. 

Currently
```
# curl "localhost:9469/probe?output=true&script=db_tablespace"
# HELP script_success Script exit status (0 = error, 1 = success).
# TYPE script_success gauge
script_success{} 0
# HELP script_duration_seconds Script execution time, in seconds.
# TYPE script_duration_seconds gauge
script_duration_seconds{} 1.788413
```
Imagining 
```
script_success{script="db_tablespace"} 0
```
And/Or optionally 
```
script_success{script="db_tablespace"} 0
# HELP Script Output param output != ignore
db_tablespace_probe_up{dbhost="dbserver",database="CUST",port="1521"} 0
```

Please let me know your thoughts on the above.
Regards,
Chris Whelan

**Comment by ricoberger** [comment]
Hi @chrisfw, I think it shouldn't be a problem to include the script name in the `script_success` metric. 

I'm using the `job` label which is added to the metric by Prometheus to indicate which script fails, maybe this is also an option for you.

If the exit status of the script is not 0, I would not "parse" the output, because this should indicate a problem with the script it self.

**Comment by chrisfw** [comment]
Hi @ricoberger , thanks for your response.  

Good point about using the job label, but I guess when testing via curl you don't have that luxury. My understanding of prometheus.yml job configuration in other scenarios is that multiple scrape targets could be included in one job definition.  I don't do this because it makes it difficult to disambiguate the metrics of which this would be a case in point, but since you are capturing the script name, it would be great just to have it included in the success metric output.
Although I am a neophyte go programmer with a Java background I could certainly make this change, but I wanted to see if you agreed with my thinking (and if you would prefer to implement it). 

I understand your point and agree with your response related to handling non-zero exit status.  My script is sort of a special case in that its sole reason to be is to generate prometheus metrics rather than being a standard script whose exit value is what gets used as a metric, but as long as the success metric (or job label) gives me what I need to discern the script, that is enough to work with.

**Comment by chrisfw** [comment]
Thank you very much for this enhancement Rico.

Regards,
Chris

**Comment by ricoberger** [comment]
Your welcome, the `script` label is available in version [v2.2.0](https://github.com/ricoberger/script_exporter/releases/tag/v2.2.0). If you have problems with that feel free to reopen the issue. If you have further suggestions please open a new issue.
